lib/pipeline.js

```diff
@@ -596,6 +627,18 @@ class PipelineModel extends BaseModel {
         return scmUri;
     }
 
+    /**
+     * Format scmUrlsWithActions object by converting scmUrl to scmUri
+     * @method _formatUrlsWithActionsStruct
+     * @param  {Object} scmUrlObject  object containing scmUrl and corresponding actions
+     * @return {Promise}
+     */
+    async _formatUrlsWithActionsStruct(scmUrlObject) {
```

Comment:
is this called anywhere ?

Reply:
Oh forgot to remove it. It's not required.